crypto/evp/evp_fetch.c

```diff
@@ -286,6 +286,18 @@ inner_evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
     return method;
 }
 
+#ifndef FIPS_MODULE
+static const char *libctx_descriptor(OPENSSL_CTX *libctx)
+{
+    return
+        openssl_ctx_is_global_default(libctx)
+        ? "Global default library context"
+        : openssl_ctx_is_default(libctx)
+        ? "Thread-local default library context"
```

Comment:
This should still be indented two spaces right.

Reply:
This code is way too clever; it's not necessary to do JAPH-style one-liners.  Even the original author can't get the indentation right.

Reply:
After all, it's just a return statement:
```
if (openssl_ctx_is_global_default(libctx))
    return "Global default library context";
if (openssl_ctx_is_default(libctx))
    return "Thread-local default library context";
return "Non-default library context";
```

Reply:
@paulidale, there's no basis in the coding style for "should", this is an unprecedented (?) case.
I'll convert this to what @richsalz proposes, that'll be the sane way to avoid a dispute.

Reply:
Done

Reply:
@richsalz's suggestion is best of all.

I'd argue that the `X ? Y : Z` in `A ? B : X ? Y : Z` is its own expression and the ?: should indent under the X.  More readable would have been writing it `A ? B : (X ? Y : Z)` where the indentation is unambiguous.

Reply:
I think this took a tern for the worse. (sorry couldn't resist..)